vendor/k8s.io/kubernetes/pkg/registry/rbac/escalation_check.go

```diff
@@ -37,7 +37,10 @@ func EscalationAllowed(ctx genericapirequest.Context) bool {
 	// system:masters is special because the API server uses it for privileged loopback connections
 	// therefore we know that a member of system:masters can always do anything
 	for _, group := range u.GetGroups() {
-		if group == user.SystemPrivilegedGroup {
+		// allow upstream group AND origin admin group to escalate
+		// TODO: remove origin case when origin has migrated to upstream RBAC
+		originSystemPrivilegedGroup := "system:cluster-admins"
```

Comment:
> This is not an authorizer check, but in the RBAC storage. Either we make our root user member of system:masters or we use this hack. It would be less intrusive to do the former in the storage test.

We plan to create a controller that syncs openshift rules to rbac rules.  That sync-ing will make this work in a steady-state cluster.  I think that's better than adding another (not normal) magic group in this particular path.